nginx/conf/nginx.conf.erb

```diff
@@ -58,7 +66,7 @@ http {
 
         location /set-cookie {
           mruby_content_handler_code '
-            var = Nginx::Var.new
+            var = Nginx::Request.new.var
             expiration = (Time.now + var.jwtap_expiration_duration_seconds.to_i).to_i
             jwt = JWT.encode({ "exp" => expiration }, Base64.decode(var.jwtap_secret_key_base64), var.jwtap_algorithm)
```

Comment:
Correct! I mentioned why in the MR description, "Removed $jwtap_algorithm while mruby-jwt only supports HS256".